angr/engines/light/engine.py

```diff
@@ -53,17 +53,20 @@ def _codeloc(self, block_only=False):
 
 class SimEngineLightVEXMixin:
 
-    def _process(self, state, successors, block=None, whitelist=None):  # pylint:disable=arguments-differ,unused-argument
+    def _process(self, state, successors, block=None, whitelist=None, visited_blocks=[]):  # pylint:disable=arguments-differ,unused-argument
```

Comment:
Do not include RDA-specific parameters into the base classes/Mixins. Use `**kwargs` instead.